linux/run.sh

```diff
@@ -35,5 +35,5 @@ popd
 
 echo Starting Tether...
 cd $DIR/../node-tuntap
-../linux/adb start-server
```

Comment:
This assumes that the user installs adb instead of using the packaged version. Is something wrong with the packaged version?

Reply:
I apologize.  This was not intended to be included in the pull request as it was pushed to my repo after the pull request was created.  Apparently I didn't understand how pull requests work.  Thanks for catching and pointing it out.  I will try to remove it.

Reply:
No worries. You can either delete and push another commit, or just `git reset HEAD^`, and `git push --force` to remove the last commit.

Reply:
I prefer not to muddy history so I'll drop the last commit and issue another pull request.  Thanks :)

Reply:
Dropping the last commit and force pushing to the same branch will simply remove the last commit. It's beautiful really...

Reply:
Done and beautiful indeed... no need to even issue another pull request.  Hopefully this works for you and others also.  Thanks for the guidance.

Reply:
Looks good to me. Thanks!

Reply:
Oh, and to answer your question about anything wrong with the packaged version of adb... Nothing wrong with adb itself, but I ran into an issue where adb did not exist at the specified path.  I think this may have been on the android device, but I didn't take the time to look into it.  Instead I changed the code to point to the system installed adb as a quick fix hack.  I'll try to properly diagnose the problem and create an issue.